protoc-gen-gogo/generator/generator.go

```diff
@@ -840,9 +840,10 @@ func (g *Generator) SetPackageNames() {
 	// Register the support package names. They might collide with the
 	// name of a package we import.
 	g.Pkg = map[string]string{
-		"fmt":   RegisterUniquePackageName("fmt", nil),
-		"math":  RegisterUniquePackageName("math", nil),
-		"proto": RegisterUniquePackageName("proto", nil),
+		"fmt":          RegisterUniquePackageName("fmt", nil),
+		"math":         RegisterUniquePackageName("math", nil),
+		"proto":        RegisterUniquePackageName("proto", nil),
+		"golang_proto": RegisterUniquePackageName("golang_proto", nil),
```

Comment:
I think we should search where g.Pkg is being used.
I mean where is the actual package set.
How do g.Pkg know that "proto" is `github.com/golang/protobuf/proto` or `github.com/gogo/protobuf/proto`